### PR TITLE
Add more prop types

### PR DESCRIPTION
**Pull request by huwshimi**
Add prop types to some components to those that were missing them.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4763/
Test PASSed.

File: jujugui/static/gui/src/app/components/added-services-list-item/added-services-list-item.js [hunk 1]
```diff
@@ -24,6 +24,11 @@ YUI.add('added-services-list-item', function() {
 
     propTypes: {
       getUnitStatusCounts: React.PropTypes.func.isRequired,
+      hovered: React.PropTypes.oneOfType([
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/added-services-list-item/added-services-list-item.js]: cool I didn't know about this option.

**Comment by hatched** [comment]
:+1:

**Comment by makyo** [comment]
:+1:

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**